key-import/src/lib.rs

```diff
@@ -50,26 +50,33 @@ impl SignersInfo {
 
 /// Secret key to be imported
 #[cfg_attr(target_arch = "wasm32", wasm_bindgen)]
-pub struct SecretKey {
+pub struct SecretScalar {
     be_bytes: zeroize::Zeroizing<Vec<u8>>,
 }
 
 #[cfg_attr(target_arch = "wasm32", wasm_bindgen)]
-impl SecretKey {
+impl SecretScalar {
     /// Parses the secret key in big-endian format (the most widely-used format)
-    ///
-    /// Throws `Error` if secret key is invalid
     #[cfg_attr(target_arch = "wasm32", wasm_bindgen(js_name = fromBytesBE))]
-    pub fn from_bytes_be(bytes: Vec<u8>) -> Result<SecretKey, Error> {
-        Ok(Self {
+    pub fn from_bytes_be(bytes: Vec<u8>) -> SecretScalar {
+        Self {
             be_bytes: bytes.into(),
-        })
+        }
+    }
+}
+
+impl SecretScalar {
+    /// Returns bytes representation of the secret key in big-endian format
+    ///
+    /// It is not exposed in WASM API
+    pub fn to_be_bytes(self) -> zeroize::Zeroizing<Vec<u8>> {
+        self.be_bytes
     }
 }
 
 /// Builds a request body that needs to be sent to Dfns API in order to import the given key.
 ///
-/// Takes as input the `secret_key` to be imported, `signers_info` (contains information
+/// Takes as input the `secret_scalar` to be imported, `signers_info` (contains information
 /// about the _n_ key holders, needs to be retrieved from Dfns API)
 /// `min_signers` (which will be the threshold and has to satisfy _2 ≤ min_signers ≤ n_),
 /// and the `protocol` and `curve` for which the imported key will be used.
```

Comment:
Maybe add a note of how a secret scalar relates to a secret key?

Reply:
Secret scalar is just a secret key represented by the scalar... I'm not sure if I write that, it'll clarify anything

Reply:
Added this: 82f4c5c

Reply:
You wrote exactly what I thought should be written here (=